From c5c9be6c48bf93cb09fd7fe1653b270b964e7a7a Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Mon, 20 May 2019 14:47:42 +0300
Subject: [PATCH 238/386] arm64: dts: 9130-crb: change PP port 2 to 2500base-t
 mode

PP port 2 connected to PHY 88E2110 and can operate on speeds
up to 2.5G. Setting  2500base-t in phy mode would allow to PHY
advertise and negotiate speed up to 2.5G.

Change-Id: Ib0c948e995c5046261d3f846347cde88ec1ae1eb
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/9270
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130-crb.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi b/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
index 7b1a504ae2c6..43dd8bb1dc9c 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
@@ -166,6 +166,6 @@
 &cp0_eth2 {
 	status = "okay";
 	phy = <&nbaset_phy0>;
-	phy-mode = "sgmii";
+	phy-mode = "2500base-t";
 	phys = <&cp0_comphy5 2>;
 };
-- 
2.17.1

